api/Response.json

```
@@ -302,6 +302,42 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "readable_byte_stream": {
+          "__compat": {
+            "description": "<code>body</code> is a readable byte stream.",
```

Periods aren't typically used in BCD feature descriptions:

```suggestion
            "description": "<code>body</code> is a readable byte stream",
```